test/unit/utils/pageData.js

```diff
@@ -0,0 +1,29 @@
+module.exports.COLLECT_PLUGIN_SOURCES = `
+<custom-tag-one src="images/sample1.png">
+  Lorem ipsum dolor sit amet, Ut enim ad minim veniam, 
+  <div data-included-from="/root/subdir/includedpage.md">
+    consectetur adipiscing elit, 
+    <div>
+      <custom-tag-two srcattr="images/sample2.png">
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      </custom-tag-two>
```

Comment:
For a sanity check, maybe have another include inside the include to ensure that it resolves correctly even if it involves 2 levels deep?